scripts/libGenerateProposals/newSuccessorMarket.js

```diff
@@ -28,24 +28,19 @@ function newSuccessorMarket(skeleton, proposalSoFar) {
 
   const result = {
     rationale: {
-      title: `Lorem Ipsum market successor`,
-      description: `A successor to nnnnn`,
+      title: `Lorem Ipsum perp`,
+      description: `A perpetual settled with an Ethereum oracle`,
```

Comment:
Successors aren't for perps - could you update this wherever it will fix the generation?

Reply:
Fixed